README.rst

```diff
@@ -91,6 +91,14 @@ Example installation command using Pathogen:
 
     git clone --recursive https://github.com/davidhalter/jedi-vim.git ~/.vim/bundle/jedi-vim
 
+Example installation using Vundle:
+
+Add the following line in you `~/.vimrc`
```

Comment:
Typo. :) `in your`.

I think `to your` would be better as well, but then again I'm not an English native.

Reply:
Yes, you're right, it's a typo.
It's fixed.

Reply:
Concerning the documentation file, it seems already alright.